### PR TITLE
Remove tally report printing from VxMark and VxMarkScan

### PR DESCRIPTION
**Pull request by adghayes**
_review commit-by-commit_

## Overview

In older versions of our hardware, we would print polls opened and polls closed reports at the precinct by saving the data onto a smartcard at VxScan, putting the card into VxMark, and printing from there. Moving forward, we'll have integrated printing in our VxScan hardware. This PR removes support for tally report printing from smartcard in VxMark and VxMarkScan. The ability to export the data from VxScan was already removed in #3748. Removing these code paths relieves a non-trivial burden of maintaining these code paths.

In addition to removing the tally report flow, we adjust some copy and styling. The new default path was formerly the unhappy path. 

## Demo Video or Screenshot

VxMark:

https://github.com/votingworks/vxsuite/assets/37960853/132b0337-58c8-4467-88b1-cadaf021210d

VxMarkScan:

https://github.com/votingworks/vxsuite/assets/37960853/cacca485-f933-4e34-9bf0-d29039935498

## Testing Plan
updated automated tests and manual pass

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by adghayes** [comment]
new modal texts, seeking feedback:

Screen Shot 2023-07-24 at 12 32 07 PM

Screen Shot 2023-07-24 at 12 24 54 PM

Screen Shot 2023-07-24 at 12 25 59 PM

Screen Shot 2023-07-24 at 12 29 22 PM

File: apps/mark/frontend/src/pages/poll_worker_screen.tsx [hunk 1]
```diff
@@ -405,14 +399,14 @@ export function PollWorkerScreen({
...
-                        />
+                      <React.Fragment key={`${pollsTransition}-button`}>
+                        <P>
+                          <UpdatePollsButton
+                            pollsTransition={pollsTransition}
+                            updatePollsState={updatePollsState}
+                            isPrimaryButton={index === 0}
+                          />
+                        </P>
                       </React.Fragment>
```
Comment (arsalansufi) [inline apps/mark/frontend/src/pages/poll_worker_screen.tsx:410]: I believe this `React.Fragment` is no longer needed. We can get rid of it and add the key to the `P` element

File: apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx [hunk 2]
```diff
@@ -417,14 +409,14 @@ export function PollWorkerScreen({
...
                     return (
-                      <React.Fragment
-                        key={`${pollsTransition}-directly-button`}
-                      >
-                        {index > 0 && ' or '}
-                        <UpdatePollsDirectlyButton
-                          pollsTransition={pollsTransition}
-                          updatePollsState={updatePollsState}
-                        />
+                      <React.Fragment key={`${pollsTransition}-button`}>
```
Comment (arsalansufi) [inline apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx:412]: Same comment here re the unneeded `React.Fragment`

File: apps/mark/frontend/src/pages/poll_worker_screen.tsx [hunk 3]
```diff
@@ -87,13 +87,13 @@ function UpdatePollsButton({
...
+        return `After polls are opened, voters will be able to mark ballots.`;
       case 'pause_voting':
-        return `After voting is paused, the machine will no longer allow voters to mark and print ballots until voting is resumed.`;
+        return `After voting is paused, voters will not be able to mark ballots until voting is resumed.`;
       case 'resume_voting':
-        return `After voting is resumed, the machine will be ready for voters to mark and print ballots.`;
+        return `After voting is resumed, voters will be able to mark ballots.`;
       case 'close_polls':
-        return `After polls are closed, the machine will no longer allow voters to mark and print ballots. Polls cannot be opened again after being closed.`;
+        return `After polls are closed, voters will no longer be able to mark ballots. Polls cannot be opened again after being closed.`;
```
Comment (arsalansufi) [inline apps/mark/frontend/src/pages/poll_worker_screen.tsx:96]: I like this permutation of the copy! Not a big deal at all but the screenshots in the PR are out-of-date now, and I was going to suggest exactly this - standardizing on "After [transition], voters [will / will not] be able to [action]." Clearer and more succinct than the "the machine will" alternatives

**Review by arsalansufi — APPROVED** [review]
LGTM! Glad this logic is gone - it was one of the more cumbersome pieces of logic to maintain as I switched us from memory cards to Java Cards 😅

File: apps/mark/frontend/src/pages/poll_worker_screen.tsx (hunk 3 above)
Comment (adghayes) [reply apps/mark/frontend/src/pages/poll_worker_screen.tsx:96]: Screenshots in the overview are correct - I just deleted the comment with the older snapshots.

File: apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx (hunk 2 above)
Comment (adghayes) [reply apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx:412]: Cool, removing.

**Comment by adghayes** [comment]
@arsalansufi I wish we could have removed it before you had to make it work with javacards!

**Pull request merged**